README.md

```diff
@@ -53,7 +53,7 @@ For guidance on installation, development, deployment, and administration, see o
 CockroachDB supports the PostgreSQL wire protocol, so you can use any available PostgreSQL client drivers to connect from various languages.
 
 - For recommended drivers that we've tested, see [Install Client Drivers](https://www.cockroachlabs.com/docs/stable/install-client-drivers.html).
-- For tutorials using these drivers, as well as supported ORMs, see [Build an App with CockroachDB](https://www.cockroachlabs.com/docs/stable/build-an-app-with-cockroachdb.html).
+- For tutorials using these drivers, as well as supported ORMs, see [Build an App with CockroachDB](https://www.cockroachlabs.com/docs/stable/hello-world-example-apps).
```

Comment:
> `For tutorials using these drivers, as well as supported ORMs, see [Build an App with CockroachDB](https://www.cockroachlabs.com/docs/stable/hello-world-example-apps).`

Could you change this to:

`For tutorials using these drivers, as well as supported ORMs, see our [Hello World Example Apps](https://www.cockroachlabs.com/docs/stable/hello-world-example-apps.html).`

Reply:
Done.